base/siteorigin-widget.class.php

```diff
@@ -424,7 +424,7 @@ function add_defaults( $form, $instance, $level = 0 ){
 				}
 			}
 			else {
-				if( !isset( $instance[$id] ) ) {
+				if( !isset( $instance[$id] ) || $instance[$id] === false ) {
```

Comment:
Same here. This is basically just sweeping a deeper issue under the rug. Default values should only be used for fields that have not been set. This also prevents a valid value of `false` from being used.